kstreams-live-update/README.md

```diff
@@ -46,7 +45,7 @@ curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json"
 # Consume aggregated messages
 
 Once you see the message "WildFly Swarm is Ready" in the logs, open the UI in the browser: http://localhost:8079/.
```

Comment:
Now: "Thorntail is Ready"

Reply:
@nilshartmann https://github.com/gunnarmorling/debezium-examples/pull/4/files